prisma-fmt/src/get_dmmf.rs

```diff
@@ -0,0 +1,24 @@
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+struct GetDmmfParams {
+    prisma_schema: String,
+}
+
+pub(crate) fn get_dmmf(params: &str) -> String {
+    let params: GetDmmfParams = match serde_json::from_str(params) {
+        Ok(params) => params,
+        Err(serde_err) => {
+            panic!("Failed to deserialize GetDmmfParams: {serde_err}");
+        }
+    };
+
+    get_dmmf_impl(params)
+}
+
+fn get_dmmf_impl(params: GetDmmfParams) -> String {
+    // Note:
+    // - if the Prisma schema is not valid, this panics
+    dmmf::dmmf_json_from_schema(&params.prisma_schema)
+}
```

Comment:
Nitpick: I find this would be more readable if the contents of the `get_dmmf_impl()` function were directly inside `get_dmmf()`. Not blocking.

Reply:
I could easily do that, but that would look different than what we did in `prisma-fmt/src/get_config.rs`, where `get_config_impl()` is defined after the public `get_config()` definition

Reply:
get_config_impl() has content, get_dmmf_impl() here is just a function that forwards to another function

Reply:
I see. Then I'll change it to

```rust
pub(crate) fn get_dmmf(params: &str) -> String {
    let params: GetDmmfParams = match serde_json::from_str(params) {
        Ok(params) => params,
        Err(serde_err) => {
            panic!("Failed to deserialize GetDmmfParams: {serde_err}");
        }
    };

    // if the Prisma schema is not valid, this panics
    dmmf::dmmf_json_from_schema(&params.prisma_schema)
}
```